IFC4x3/Sections/Resource definition data schemas/Schemas/IfcGeometricModelResource/Entities/IfcVoxelGrid/DocEntity.xml

```diff
@@ -36,8 +36,7 @@ along X, then Y and finally Z.</Documentation>
 	<WhereRules>
 		<DocWhereRule Name="CorrectGridSize" UniqueId="d21d238c-a7bf-4473-9503-e1f72b3552c2">
 			<Documentation>The size of the grid (_Voxels_ array) shall be equal to the product of the number of voxels in X, Y and Z directions.</Documentation>
-			<Expression>SIZEOF(SELF\Voxels) = NumberOfVoxelsX * NVL(NumberOfVoxelsY, NumberOfVoxelsX) * NVL(NumberOfVoxelsZ, NumberOfVoxelsX)</Expression>
+			<Expression>SIZEOF(Voxels) = NumberOfVoxelsX * NVL(NumberOfVoxelsY, NumberOfVoxelsX) * NVL(NumberOfVoxelsZ, NumberOfVoxelsX)</Expression>
```

Comment:
Is the NVL thingy really needed now since they are all mandatory?

Reply:
Nope, not anymore if we agree on the mandatory nature of these attributes.

I prefer making them mandatory, as stated in the PR https://github.com/bSI-InfraRoom/IFC-Specification/pull/809. What do you think?

Reply:
> Nope, not anymore if we agree on the mandatory nature of these attributes.
> 
> I prefer making them mandatory, as stated in the PR #809. What do you think?

Agree. Imho doesn't add much regarding space or inconvenience and requires exports to always consider all dimensions.